### ISSUE TITLE
Unable to run after setup

### ISSUE THREAD

**Issue by manparvesh** [yoda-pa/yoda#28]
What are the steps to reproduce this issue?
-------------------------------------------
1. Install `yoda` according to the steps mentioned in README (without using --editable while installing)

What happens?
-------------
```
Traceback (most recent call last):
  File "/home/mp/Desktop/github_repos/yoda/venv/bin/yoda", line 11, in <module>
    load_entry_point('yoda==0.1.0', 'console_scripts', 'yoda')()
  File "/home/mp/Desktop/github_repos/yoda/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 570, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/mp/Desktop/github_repos/yoda/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2751, in load_entry_point
    return ep.load()
  File "/home/mp/Desktop/github_repos/yoda/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2405, in load
    return self.resolve()
  File "/home/mp/Desktop/github_repos/yoda/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2411, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/mp/Desktop/github_repos/yoda/venv/local/lib/python2.7/site-packages/yoda.py", line 6, in <module>
    from modules import *
  File "/home/mp/Desktop/github_repos/yoda/modules/learn.py", line 30, in <module>
    for line in pkgutil.get_data('yoda', 'resources/vocab-words.txt').split('\n'):
  File "/usr/lib/python2.7/pkgutil.py", line 591, in get_data
    return loader.get_data(resource_name)
  File "/usr/lib/python2.7/pkgutil.py", line 255, in get_data
    return open(pathname, "rb").read()
IOError: [Errno 2] No such file or directory: '/home/mp/Desktop/github_repos/yoda/venv/local/lib/python2.7/site-packages/resources/vocab-words.txt'
```

What were you expecting to happen?
----------------------------------
Should work properly

What versions of software are you using?
----------------------------------------
**Operating System:** Ubuntu 16.04
**yoda:** 0.1.0

**Comment by manparvesh** [comment]
This is probably related to #7

**Comment by manparvesh** [comment]
We need to know how to save the files in the resources folder can be saved to the folder where this package is installed. In this case, `<location>/yoda/venv/local/lib/python2.7/site-packages/`

**Comment by eyalhei** [comment]
Not sure how this works, But I like to give it a try

**Comment by manparvesh** [comment]
fixed in #32
